hydrolib/core/basemodel.py

```diff
@@ -744,6 +723,101 @@ def sync_post(model: BaseModel, acc: FileLoadContext) -> FileLoadContext:
             context.push_new_parent(self._absolute_anchor_path, self._relative_mode)
             traverser.traverse(self, context)
 
+    @property
+    def _relative_mode(self) -> ResolveRelativeMode:
+        """Gets the ResolveRelativeMode of this FileModel.
```

Comment:
Docstrings should be in imperative form, right? --> "Get the Resolve[..]"
also in some other places.